hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/cache/CacheManager.java

```diff
@@ -56,43 +56,23 @@
     private final Logger log = Logger.getLogger(CacheManager.class);
     private final MsgLogger msgLog = MsgLogger.LOGGER;
 
-    private static final String PUBLISH_REQUIRE_PREFIX_PROP = "hawkular-alerts.publish-require-prefix";
-    private static final String PUBLISH_REQUIRE_PREFIX_ENV = "PUBLISH_REQUIRE_PREFIX";
     private static final String DISABLE_PUBLISH_FILTERING_PROP = "hawkular-alerts.disable-publish-filtering";
     private static final String DISABLE_PUBLISH_FILTERING_ENV = "DISABLE_PUBLISH_FILTERING";
     private static final String RESET_PUBLISH_CACHE_PROP = "hawkular-alerts.reset-publish-cache";
     private static final String RESET_PUBLISH_CACHE_ENV = "RESET_PUBLISH_CACHE";
 
-    private static final Set<String> DATA_ID_PREFIXES = new HashSet<>();
-
-    static {
-        // TODO: For now just use the built-in H Metrics prefixes as a OOB convenience, but in the
-        //       future this should maybe be configurable where a list of prefixes could be supplied
-        //       via system prop.
-        DATA_ID_PREFIXES.add("hm_a_");  // Metrics AVAILABILITY
```

Comment:
Do you remember if these prexises are documented in the metrics.ear code ?
Just wondering for documentation purposes.
It could be nice to have an interface with swagger to maintain them in some way.
I remember I introduced one in the preliminar code but with the merging of m+a not sure if this is missing somewhere.
Usecase: to reference an external user the correct syntax on triggers creation.